integrations/jira/main.py

```diff
@@ -75,6 +113,14 @@ async def handle_webhook_request(data: dict[str, Any]) -> dict[str, Any]:
         logger.info(f'Received webhook event for issue: {data["issue"]["key"]}')
         issue = await client.get_single_issue(data["issue"]["key"])
         await ocean.register_raw(ObjectKind.ISSUE, [issue])
+    elif "board" in data:
+        logger.info(f'Received webhook event for board: {data["board"]["id"]}')
+        board = await client.get_single_board(data["board"]["id"])
```

Comment:
You can reuse the get_single_resource() function instead of defining individual functions for all the kinds

Reply:
The get_single_resource method is quite vague and requires the URL to be compiled before being used. If I use that directly here, this function will look more complex with the if and else branches already there